vendor/engines/split_accounts/app/models/split_account.rb

```diff
@@ -0,0 +1,28 @@
+class SplitAccount < Account
+
+  has_many :splits, foreign_key: :parent_split_account_id, inverse_of: :parent_split_account
+
+  validate :valid_percent_total
+  validate :one_split_has_extra_penny
+
+  def valid_percent_total
+    unless percent_total == 100
```

Comment:
This is a style thing of mine, but I prefer not to use `unless` except in one liners where it's easier to read than an `if !`. I would probably change this to `if percent_total != 100`.